### PR TITLE
HTML page with catalog created

### PR DESCRIPTION
**Pull request by roman-zherebetskyi**
- [DEMO LINK](https://roman-zherebetskyi.github.io/layout_catalog/)
- [TEST REPORT LINK](https://roman-zherebetskyi.github.io/layout_catalog/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -1,12 +1,455 @@
...
-    <h1>Catalog</h1>
+    <header class="header">
+      <a class="header__logo" href="#logo">
+        <img
+          src="/images/logo.png"
+          alt="logo.png">
+      </a>
+
+      <nav class="nav">
+        <div class="nav__list">
```
Comment (grafit-off) [inline src/index.html:28]: You can use semantic tag ul for lists

File: src/index.html [hunk 2]
```diff
@@ -1,12 +1,455 @@
...
+
+          <div class="card__price-container">
+            <p class="card__price-label">
+              Price:
+            </p>
+
+            <p class="card__price">
+              $2,199
+            </p>
+          </div>
```
Comment (grafit-off) [inline src/index.html:106]: It's not two paragraph, it's only one. (Приклад: "ціна 2000 грн", це не два різні параграфа, це по суті одне речення, не можна написати: "Ціна.  2000 грн.", бо це одне речення, отак і з параграфами)
```suggestion
          <p class="card__price-container">
            <span class="card__price-label">
              Price:
            </span>

            <span class="card__price">
              $2,199
            </span>
          </p>
```

File: src/styles/blocks/card.scss [hunk 3]
```diff
@@ -0,0 +1,97 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  width: $card-width;
+  padding: 32px 16px 16px;
+  margin-bottom: 46px;
+  margin-right: 48px;
```
Comment (grafit-off) [inline src/styles/blocks/card.scss:7]: Card it's a BEM block, BEM block can't have margins!!!

File: src/styles/blocks/card.scss [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+    font-size: 16px;
+    color: $c-text-main;
+    font-weight: 700;
+  }
+
+  &__buy-button {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 40px;
```
Comment (grafit-off) [inline src/styles/blocks/card.scss:78]: To my opinion it's not good to set fixed height, we can reach needed sizes with paddings, without limiting our height. But in this case it's not critical.

File: src/styles/blocks/card.scss [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+    line-height: 16px;
+    color: $c-white;
+    font-weight: 700;
+
+    text-decoration: none;
+    background: $c-blue;
+    border: 1px solid $c-blue;
+    border-radius: 5px;
+    text-transform: uppercase;
+    transition: 0.2s;
```
Comment (grafit-off) [inline src/styles/blocks/card.scss:90]: You should add transition property! if you don't, the browser will watch to all properties that can change, its not good for performance

File: src/styles/blocks/catalog.scss [hunk 6]
```diff
@@ -0,0 +1,7 @@
+.catalog {
+  display: flex;
+  flex-wrap: wrap;
+  padding: 50px 40px;
+  margin: 0 auto;
+  width: 1024px;
```
Comment (grafit-off) [inline src/styles/blocks/catalog.scss:6]: CHECKLIST [LAYOUT] - don't hardcode max-width, your catalog container should work fine for any number of cards.
You should  use variables and some calculating for max-width

File: src/styles/blocks/nav.scss [hunk 7]
```diff
@@ -0,0 +1,46 @@
...
+    align-items: center;
+    margin-left: 20px;
+    padding: 23px 0;
+
+    line-height: 14px;
+    text-transform: uppercase;
+    text-decoration: none;
+    font-weight: 500;
+    color: $c-text-main;
+    transition-duration: 0.2s;
```
Comment (grafit-off) [inline src/styles/blocks/nav.scss:22]: You should add transition property! if you don't, the browser will watch to all properties that can change, its not good for performance

**Review by grafit-off — APPROVED** [review]
Good job, keep going 👍 ! Left some comments below!

File: src/index.html (hunk 1 above)
Comment (roman-zherebetskyi) [reply src/index.html]: you're right! fixed and updated

File: src/index.html (hunk 2 above)
Comment (roman-zherebetskyi) [reply src/index.html]: again semantic mistakes... fixed and updated!

File: src/styles/blocks/card.scss (hunk 3 above)
Comment (roman-zherebetskyi) [reply src/styles/blocks/card.scss]: rude mistake fixed and updated!

File: src/styles/blocks/nav.scss (hunk 7 above)
Comment (roman-zherebetskyi) [reply src/styles/blocks/nav.scss:22]: yeah you already told me, okay I'll try to be more attentive next time!
thank you for the review!!! 🤝

File: src/styles/blocks/catalog.scss (hunk 6 above)
Comment (roman-zherebetskyi) [reply src/styles/blocks/catalog.scss]: rude mistake fixed and updated! tnx for noticing!

File: src/styles/blocks/card.scss (hunk 5 above)
Comment (roman-zherebetskyi) [reply src/styles/blocks/card.scss:88]: you mean is better to use transition-duration?
I hope it's not really a big deal?

File: src/styles/blocks/card.scss (hunk 4 above)
Comment (roman-zherebetskyi) [reply src/styles/blocks/card.scss:76]: it's height of the button, let it be, later I can use the button as an independent block
btw: in my opinion is better to use in instead to, you can check it 😉

File: src/styles/blocks/card.scss (hunk 5 above)
Comment (grafit-off) [reply src/styles/blocks/card.scss:88]: No, e.g.:
transition: border 0.2s;

Comment (roman-zherebetskyi) [reply src/styles/blocks/card.scss:88]: okay okay now I got it, thank you!

File: src/styles/utilits/_variables.scss [hunk 8]
```diff
@@ -0,0 +1,11 @@
+
+$card-width: 200px;
+$card-margin-right: 48px;
+$card-margin-bottom: 46px;
+$catalog-container-width: ($card-width * 4) + ($card-margin-right * 3);
```
Comment (MaksimusCRS) [inline src/styles/utilits/_variables.scss:5]: Remove magic numbers, add variables instead

File: src/styles/blocks/catalog.scss [hunk 9]
```diff
@@ -0,0 +1,24 @@
+.catalog {
+
+  padding: 50px 0;
+  margin: 0 auto;
```
Comment (MaksimusCRS) [inline src/styles/blocks/catalog.scss:4]: Remove margin from BEM block

File: src/styles/blocks/card.scss (hunk 5 above)
Comment (MaksimusCRS) [reply src/styles/blocks/card.scss:88]: Add transition property

**Review by MaksimusCRS — CHANGES_REQUESTED** [review]

File: src/styles/blocks/card.scss (hunk 5 above)
Comment (roman-zherebetskyi) [reply src/styles/blocks/card.scss:88]: fixed

File: src/styles/blocks/catalog.scss (hunk 9 above)
Comment (roman-zherebetskyi) [reply src/styles/blocks/catalog.scss:4]: fixed

File: src/styles/utilits/_variables.scss (hunk 8 above)
Comment (roman-zherebetskyi) [reply src/styles/utilits/_variables.scss:5]: I did, but this is my variables, now I add more variables

**Review by MaksimusCRS — APPROVED** [review]
GJ

**Pull request closed without merging**